src/bundle/Resources/public/js/scripts/admin.content.edit.js

```diff
@@ -71,6 +80,25 @@
                     }, new Set())
             ).join();
 
+            // Hide warnings icons
+            doc.querySelectorAll(`.nav-item .ez-warning-icon`).forEach((node) => {
+                node.style.display = 'none';
+            });
+
+            // Show warnings icons for tabs with invalide fields
+            validationResults
+                .reduce((total, result) => {
+                    result.tabErroKeys.map((key) => {
+                        total.push(key);
+                    });
+
+                    return total;
+                }, [])
+                .filter((value, index, self) => self.indexOf(value) === index)
+                .forEach((key) => {
+                    doc.querySelector(`.nav-item--${key} .ez-warning-icon`).style.display = 'block';
```

Comment:
1. We should not add modifiers to the bootstrap classes, please add our class with prefix `ez-`
2. Let's make it consistent with other places and add to the tab item class with modifier `--invalid` which will show the warning icon